client/app/lib/components/layouts/Section.tsx

```diff
@@ -0,0 +1,60 @@
+import { ReactNode } from 'react';
+import { Grid, Typography, Divider, Container } from '@mui/material';
+import { SxProps, Theme } from '@mui/material/styles';
+
+interface SectionProps {
+  title?: string;
+  subtitle?: string;
+  children?: ReactNode;
+  sticksToNavbar?: boolean;
+}
+
+const styles: { [key: string]: SxProps<Theme> } = {
+  marginBottom: {
+    marginBottom: 2,
+  },
+  content: {
+    '> *:not(:last-child)': { marginBottom: 2 },
+  },
+  container: {
+    '& + &': { marginTop: 2 },
+  },
+};
+
+const Section = (props: SectionProps): JSX.Element => {
+  const stickyHeadersLgOnly: SxProps<Theme> = (theme) => ({
+    [theme.breakpoints.up('lg')]: {
+      position: 'sticky',
+      top: props.sticksToNavbar ? '5rem' : '-1em',
+      alignSelf: 'flex-start',
+    },
+  });
+
+  return (
+    <Container disableGutters maxWidth="lg" sx={styles.container}>
```

Comment:
Actually this was an intended behaviour. Without `maxWidth`, in wide screens, the form will grow and we end up with textboxes that are awfully long and description texts that may be difficult to read. [This article explains why one should limit a body's width for readability](https://baymard.com/blog/line-length-readability).

At first it looks weird too, having the big 'Edit Assessment' title and the form not aligning, but apparently GitLab does this too in their User Settings page.

![image](https://user-images.githubusercontent.com/51525686/187836356-9ef24aad-85a1-4313-94dc-c36d7cab589d.png)

Maybe we can move the 'Edit Assessment' title and Save button to align to the middle of the form too. This is actually better because all the important elements are clustered together in wide screens.

Reply:
Got it thanks!